hooks/queries/votes/usePastVotes.ts

```diff
@@ -2,15 +2,22 @@ import { useQuery } from "@tanstack/react-query";
 import { pastVotesKey } from "constants/queryKeys";
 import { getPastVotes } from "graph";
 import { useHandleError, useVoteTimingContext } from "hooks";
+import { usePaginationContext } from "hooks/contexts/usePaginationContext";
 
 export function usePastVotes() {
   const { roundId } = useVoteTimingContext();
   const onError = useHandleError();
+  const {
+    pageStates: { pastVotesPage },
+  } = usePaginationContext();
 
-  const queryResult = useQuery([pastVotesKey, roundId], () => getPastVotes(), {
+  const { resultsPerPage, number } = pastVotesPage;
+
+  const queryResult = useQuery([pastVotesKey, roundId, pastVotesPage], () => getPastVotes(resultsPerPage, number), {
```

Comment:
by adding `pastVotesPage` as a query key, we tell react query to re-run the query with the new page args whenever the values in `pastVotesPage` change